packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -211,8 +216,15 @@ export class AzureCommunicationCallWithChatAdapter implements CallWithChatAdapte
     await this.callAdapter.leaveCall();
   }
   /** Start a new Call. */
-  public startCall(participants: string[], options?: StartCallOptions): Call | undefined {
-    return this.callAdapter.startCall(participants, options);
+  public startCall(participants: string[] | CommunicationIdentifier[], options?: StartCallOptions): Call | undefined {
+    const communicationParticipants = participants.map((participant) => {
```

Comment:
this conversion logic is in many different places, consider a single function
```
const participantToCommunicationIdentifier = (participant: string | CommunicationIdentifier) =>  typeof participant === 'string' ? fromFlatCommunicationIdentifier(participant) : participant;
```

then this can nicely become 
```suggestion
    const communicationParticipants = participants.map(participantToCommunicationIdentifier);
```

Reply:
Created a common function in acs-ui-common and used it everywhere. LMKWYT